manager_D3S.py

```diff
@@ -10,12 +10,15 @@
 import signal
 import sys
 from Crypto.Cipher import AES
+from collections import deque
```

Comment:
So the current behavior on `self.wqueue1`, using `append` and `pop`, does not require a `deque`, you can do the same thing with a plain list.

Unless you want to keep the queue from growing too large, then `deque` has a handy initialization parameter `maxlen` so it can throw away the oldest values in an efficient way.